src/StackNavigation.Uno.WinUI/StackNavigation.Uno.WinUI.csproj

```diff
@@ -13,38 +12,37 @@
     <Description>Unified cross-platform tools to perform ViewModel-based navigation.</Description>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageReadmeFile>README.md</PackageReadmeFile>
-  	<PackageTags>navigation;mvvm;xamarin;ios;android;uwp;chinook</PackageTags>
-		<PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
-		<PackageProjectUrl>https://github.com/nventive/Chinook.Navigation</PackageProjectUrl>
-
-		<PublishRepositoryUrl>true</PublishRepositoryUrl>
-		<EmbedUntrackedSources>true</EmbedUntrackedSources>
-		<IncludeSymbols>true</IncludeSymbols>
-		<SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <PackageTags>navigation;mvvm;maui;winui;net6.0-ios;net6.0-android;chinook</PackageTags>
```

Comment:
for the **tags**, I think **ios** and **android** are better than **net6.0-ios** and **net6.0-android** (which are target framework monikers)